packages/web-components/src/wallet-connection/watchWallet.js

```diff
@@ -90,7 +90,7 @@ export const watchWallet = async (chainStorageWatcher, address) => {
       );
     })
   ).catch(() => {
-    throw Error(Errors.noSmartWallet);
+    console.error(Errors.noSmartWallet);
```

Comment:
I think instead of awaiting this entire promise starting on line 69, we just want to watch the path with line 71-90. Then there's a question of how to detect/notify if the smart wallet doesn't exist. I think we should make a separate notifier like `smartWalletStatusNotifier`, initial value of `null`, and in line 85, instead of `rej()`, update it with something like `{provisioned: false}`. And instead of `res()` in line 75, update it with something like `{provisioned: true}`. Then expose that notifier in the return value along with the others so the dapp knows explicitly whether it's provisioned.

Reply:
The overall idea is that we set up all the watchers regardless of whether the smart wallet exists yet, never throw, and simply update `smartWalletStatusNotifier` along with the other notifiers whenever the data comes in.

Reply:
have updated it such that the user is notified of the status of provisioning and not erroring out